Formula/tut.rb

```diff
@@ -0,0 +1,18 @@
+class Tut < Formula
+  desc "TUI for Mastodon with vim inspired keys"
+  homepage "https://tut.anv.nu"
+  url "https://github.com/RasmusLindroth/tut/archive/refs/tags/1.0.21.tar.gz"
+  sha256 "64139c3e8f5873d78e0044f158bf9070b18e0a96cb7ae48a350d75476d9073e9"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"tut"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/tut --version")
+    assert_match "TUI for Mastodon", shell_output("#{bin}/tut --help")
```

Comment:
we might need a better test for this. You can find similar UI tests in the repo.

Reply:
Thanks for the feedback. I went ahead and tested the `tut --new-user` command that prompts for the mastodon instance. From there you have to authenticate with a browser and that flow is fairly complicated, but this proves that the auth flow starts to work.

Reply:
That is good! 👍